cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -54,7 +54,7 @@ public static Secret findSecretWithName(List<Secret> secrets, String sname) {
     }
 
     public static int getCertificateValidity(CertificateAuthority certificateAuthority) {
-        int validity = AbstractModel.CERTS_EXPIRATION_DAYS;
+        int validity = AbstractModel.CERTS_VALIDITY_DAYS;
         if (certificateAuthority != null
                 && certificateAuthority.getValidityDays() > 0) {
```

Comment:
Since you change it from primitive to boxed, there is now the possibility of an NPE here (though perhaps is never happens in practice).